### PR TITLE
Issue 10

### PR DESCRIPTION
**Pull request by bhavika24**
https://github.com/enzoftware/hotel_booking_app/issues/10

feat: Running test on Github actions

### CODE REVIEW

File: .github/workflows/flutter-ci.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  build:
+    runs-on: [windows-latest]
+    steps:
+    
+    # Setup Java environment in order to build the Android app.
+    - uses: actions/checkout@v1
+    - uses: actions/setup-java@v1
+      with:
+        java-version: '13.x'
+
```
Comment (enzoftware) [inline .github/workflows/flutter-ci.yml:16]: Add `on: pull_request` to run this action every time a new pull request arrives

**Comment by enzoftware** [comment]
Looks good, just add the line in the comment and it'll be accepted.

File: .github/workflows/flutter-ci.yml (hunk 1 above)
Comment (enzoftware) [reply .github/workflows/flutter-ci.yml:16]: https://medium.com/mobile-development-group/github-actions-for-flutter-cf02923d7b5d

**Comment by bhavika24** [comment]
@enzoftware I have updated the action to run on pull requests. Please review it.

**Comment by enzoftware** [comment]
LGTM! Thanks!

**Pull request merged**